### PR TITLE
Patch 4

### PR DESCRIPTION
**Pull request by toraritte**
Thank you for all the great work!

### CODE REVIEW

File: lib/ecto.ex [hunk 1]
```diff
@@ -355,7 +355,7 @@ defmodule Ecto do
 
   ### Embeds
 
-  Ecto also supports embeds. While associations keep parent and child
+  Ecto also supports [embeds](https://docs.mongodb.com/manual/core/data-model-design/#data-modeling-embedding). While associations keep parent and child
```
Comment (josevalim) [inline lib/ecto.ex]: Embeds are not specific to mongo, so I don't think we should link exclusively to mongo. :)

**Comment by josevalim** [comment]
I have added one comment, thank you!

File: lib/ecto.ex (hunk 1 above)
Comment (toraritte) [inline lib/ecto.ex]: Yeah, I had my concerns about that too but I couldn't find any DB-agnostic description of it such as a wikipedia entry. Thanks again and happy new year!

File: lib/ecto.ex [hunk 2]
```diff
@@ -389,7 +389,7 @@ defmodule Ecto do
   Ecto requires developers to specify the key `:ecto_repos` in their application
   configuration before using tasks like `ecto.create` and `ecto.migrate`. For example:
 
-      config :my_app, :ecto_repos, [MyApp.Repo]
+      config :my_app, ecto_repos: [MyApp.Repo]
```
Comment (josevalim) [inline lib/ecto.ex]: Btw, those two calls are equivalent. Both would work. :)

**Comment by josevalim** [comment]
Merged one commit manually, thank you!

**Pull request closed without merging**